### DISCUSSION TITLE
\maketitle error

### DISCUSSION THREAD

**Discussion by urdusentimentanalysis** [whutug/whu-thesis#299] [Show and tell]
3. The \filcenter command on line 279 of WHUPhd.cls has never been defined. If you had added this command yourself, it is best to revert to the original settings provided by WHUPhd.cls i.e. do not edit things yourself (or seek advice from the template maintainer).

4. On lines 87, 88 of WHYPhd.cls: there must be something to typeset between the \ziju{1} and \\[3mm] otherwise you will get a "there's no line here to break" error; you cannot remove both 武汉大学 and the image. (Again, do not edit the .cls file yourself without first asking the template author's advice.)
  I got these errors in my  overleaf latex file for phdwhu i have contacted the overleaf support but they also unable to answer I have \maketitle error also which make print only Chinese title page only. It worth mentiong that we are using urdu language packages
![1730314250753](https://github.com/user-attachments/assets/e9941a9f-9de1-4316-bfcd-f6fe0e1794d9)
 using \bale command which creating issues but with the help of team its solved but these issues we are unable to fix.

**Comment by SwitWu** [comment]
Hi. You are definitely opening a discussion at the wrong place since the class file you're using does not belong to this repo.
